recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,15 @@
+cuda_compiler:
+- nvcc
+cuda_compiler_version:
+- ""
```

Comment:
This let's conda-build (and therefore conda) fill this with whatever version it wants so it can complete the recipe parsing phases. However as [we exclude using the `cuda` compiler in this case]( https://github.com/conda-forge/ucx-split-feedstock/pull/14/files#diff-e178b687b10a71a3348107ae3154e44cR42 ), it is irrelevant what version was used. Thus we are able to turn this into the CPU-only case.

Reply:
For context, [this was originally `None`]( ), but that [caused problems]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=73989 ) as conda-build still wanted to find `nvcc_linux-64` with version `None` despite it not being used anywhere except as part of rendering. Exception included below.

<details>
<summary>Exception:</summary>

```python
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/conda_build/environ.py", line 756, in get_install_actions
    actions = install_actions(prefix, index, specs, force=True)
  File "/opt/conda/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/opt/conda/lib/python3.7/site-packages/conda/plan.py", line 474, in install_actions
    txn = solver.solve_for_transaction(prune=prune, ignore_pinned=not pinned)
  File "/opt/conda/lib/python3.7/site-packages/conda/core/solve.py", line 117, in solve_for_transaction
    should_retry_solve)
  File "/opt/conda/lib/python3.7/site-packages/conda/core/solve.py", line 158, in solve_for_diff
    force_remove, should_retry_solve)
  File "/opt/conda/lib/python3.7/site-packages/conda/core/solve.py", line 275, in solve_final_state
    ssc = self._add_specs(ssc)
  File "/opt/conda/lib/python3.7/site-packages/conda/core/solve.py", line 555, in _add_specs
    explicit_pool = ssc.r._get_package_pool(self.specs_to_add)
  File "/opt/conda/lib/python3.7/site-packages/conda/resolve.py", line 523, in _get_package_pool
    pool = self.get_reduced_index(specs)
  File "/opt/conda/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/opt/conda/lib/python3.7/site-packages/conda/resolve.py", line 544, in get_reduced_index
    explicit_specs, features = self.verify_specs(explicit_specs)
  File "/opt/conda/lib/python3.7/site-packages/conda/resolve.py", line 280, in verify_specs
    raise ResolvePackageNotFound(bad_deps)
conda.exceptions.ResolvePackageNotFound: 
  - nvcc_linux-64=None

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 445, in main
    execute(sys.argv[1:])
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 436, in execute
    verify=args.verify, variants=args.variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 2343, in build_tree
    notest=notest,
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 1334, in build
    output_metas = expand_outputs([(m, need_source_download, need_reparse_in_env)])
  File "/opt/conda/lib/python3.7/site-packages/conda_build/render.py", line 746, in expand_outputs
    for (output_dict, m) in _m.copy().get_output_metadata_set(permit_unsatisfiable_variants=False):
  File "/opt/conda/lib/python3.7/site-packages/conda_build/metadata.py", line 2047, in get_output_metadata_set
    bypass_env_check=bypass_env_check)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/metadata.py", line 721, in finalize_outputs_pass
    permit_unsatisfiable_variants=permit_unsatisfiable_variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/render.py", line 527, in finalize_metadata
    exclude_pattern)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/render.py", line 390, in add_upstream_pins
    permit_unsatisfiable_variants, exclude_pattern)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/render.py", line 378, in _read_upstream_pin_files
    permit_unsatisfiable_variants=permit_unsatisfiable_variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/render.py", line 154, in get_env_dependencies
    channel_urls=tuple(m.config.channel_urls))
  File "/opt/conda/lib/python3.7/site-packages/conda_build/environ.py", line 758, in get_install_actions
    raise DependencyNeedsBuildingError(exc, subdir=subdir)
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {'nvcc_linux-64=None'}
```

</details>

Reply:
Have changed this to use `None`, which now works after ironing out some issues.